crates/rethnet_evm_napi/src/state.rs

```diff
@@ -114,15 +125,19 @@ impl StateManager {
 
     /// Creates a state checkpoint that can be reverted to using [`revert`].
     #[napi]
+    #[cfg_attr(feature = "tracing", tracing::instrument)]
     pub async fn checkpoint(&self) -> napi::Result<()> {
         self.state
             .checkpoint()
             .await
-            .map_err(|e| napi::Error::new(Status::GenericFailure, e.to_string()))
+            .map_err(|e| napi::Error::new(Status::GenericFailure, e.to_string()))?;
+
+        Ok(())
```

Comment:
Is this necessary? This seems redundant compared with the previous version, which still compiles fine AFAICT.

Reply:
You're right. It's not necessary. The old code was fine

Reply:
You're right. It's not necessary. The old code was fine